docs/user/troubleshooting-guides/03-00-cannot-connect-to-service/v1beta2/03-40-api-rule-troubleshooting.md

```diff
@@ -0,0 +1,162 @@
+# Issues When Creating an APIRule Custom Resource in Version v1beta2
+
+## Symptom
+
+When you create an APIRule custom resource (CR), an instant validation error appears, or the APIRule CR has the `ERROR` status, for example:
+
+```bash
+kubectl get apirules httpbin
+
+NAME      STATUS   HOST
+httpbin   ERROR    httpbin.xxx.shoot.canary.k8s-hana.ondemand.com
+```
+
+The error may signify that your APIRule CR is in an inconsistent state and the service cannot be properly exposed.
+To check the error message of the APIRule CR, run:
+
+
+```bash
+kubectl get apirules -n <namespace> <api-rule-name> -o=jsonpath='{.status.description}'
+```
+
+---
+## The **issuer** Configuration of **jwt** Access Strategy Is Missing
+### Cause
+
+The following APIRule is missing the **issuer** configuration for the **jwt** access strategy:
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      jwt:
+```
+
+If your APIRule is missing the **issuer** configuration for the **jwt** access strategy, the following error appears:
+
+```
+{"code":"ERROR","description":"Validation error: Attribute \".spec.rules[0].jwt\": supplied config cannot be empty"}
+```
+
+### Remedy
+
+Add JWT configuration for the **issuer** or ``. Here's an example of a valid configuration:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      jwt:
+        authentications:
+          - issuer: "https://dev.kyma.local"
+            jwksUri: "https://example.com/.well-known/jwks.json"
+```
+
+## Invalid **issuer** for the **jwt** Access Strategy
+### Cause
+
+Here's an example of the APIRule CR with an invalid **issuer** URL configured:
+
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      jwt:
+        authentications:
+          - issuer: ://unsecured.or.not.valid.url
+            jwksUri: https://example.com/.well-known/jwks.json
+```
+
+If the **issuer** contains `:`, it must be a valid URI. Otherwise, you get the following error, and the APIRule CR is not created:
```

Comment:
Is it `URI` or `URL`? I'm asking because you mention `URL` in the error message.

Reply:
thanks for spotting this
here's an issue for updating the error message: https://github.com/kyma-project/api-gateway/issues/1102